addons/im_livechat/tests/test_message.py

```diff
@@ -92,7 +96,7 @@ def test_message_format(self):
             'res_id': channel_livechat_1.id,
             'scheduledDatetime': False,
             'sms_ids': [],
-            'starred_partner_ids': [],
+            'starredPersonas': [],
```

Comment:
`starred_partner_ids` is a field name in python, I would not rename it (this is the inverse of what we're trying to do)

Reply:
But all this stuff makes no sense anyway, we should never receive the other persons that have starred it.

This should be "personalize" to only the current user, never sent on the bus.

Reply:
Idea was to keep `starred_partner_ids` in case the mobile app might still need it with current formatting.
But if we just drop it, then we can reuse name.
IMO it's good to reuse python model names, but we should not blindly accept poor naming like suffixes `_id(s)`, and also "partner" model is not the same as "persona" model, the JS models are more mature than python models in that regards.

So I'd say `starredPersonas` is better than `starred_partner_ids`.